framework/helpers/BaseStringHelper.php

```diff
@@ -338,4 +338,35 @@ public static function base64UrlDecode($input)
     {
         return base64_decode(strtr($input, '-_', '+/'));
     }
+
+    /**
+     * Masks a token to make it uncompressible.
+     * This does not secure the token since the encryption key is prepended to the result.
+     * @param string $token An unmasked token.
+     * @return string A masked token.
+     * @since 2.0.13
+     */
+    public static function maskToken($token)
```

Comment:
What's the use case for these except masking CSRF-token before outputting it to a browser? Could it be actually used by end user?

If answer is no, I'd move it back to security as private methods.